netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1241,6 +1276,13 @@ _net_ebpf_ext_process_redirect_verdict(
             }
             commit_layer_data = TRUE;
 
+            if (_net_ebpf_ext_sock_addr_is_connection_locally_redirected_by_others(connect_request, filter->filterId)) {
```

Comment:
I think we should add a trace in this case, as this is a scenario of multiple callouts, and a possibility of interference between the 2.